dataland-e2etests/src/test/kotlin/org/dataland/e2etests/MetaDataControllerTest.kt

```diff
@@ -62,9 +78,8 @@ class MetaDataControllerTest {
         val numberOfDataSetsToPostPerCompany = 3
         val totalNumberOfDataSets = numberOfCompanies * numberOfDataSetsToPostPerCompany
         val initialSizeOfDataMetaInfoList = metaDataControllerApi.getListOfDataMetaInfo("", "").size
-
         val listOfTestCompanyIds =
-            populateDatalandWithCompaniesAndEuTaxnomyDataSets(numberOfCompanies, numberOfDataSetsToPostPerCompany)
+            createCompaniesAndEuTaxonomyDataSets(numberOfCompanies, numberOfDataSetsToPostPerCompany)
```

Comment:
Dieser gesamte Test ist nicht so gelungen. Es gibt hier multiple asserts weil ja in der Tat viele verschiedene Dinge getestet werden. Da könnte man darüber nachdenken das auseinander zu nehmen.

Reply:
_Beginne es mir anzuschauen_

Reply:
Du hast recht.  Dann fange ich an den Test zu splitten.

Reply:
_Beginne Test zu splitten_

Reply:
Test wurde gesplittet. fixed?